mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -696,6 +800,63 @@ def log_param(self, run_id, param):
                 else:
                     raise
 
+    def _log_params(self, run_id, params):
+        if not params:
+            return
+        # eliminate duplicate Param entities
+        param_instances = {}
+        for param in params:
+            if param.key not in param_instances:
+                param_instances[param.key] = SqlParam(
+                    run_uuid=run_id, key=param.key, value=param.value
+                )
+        param_instances = list(param_instances.values())
+
+        # use SessionManager instead of ManagedSessionManager to set
+        # isolation level for individual transaction.
+        # Here, we open a connection with SERIALIZABLE isolation level.
+        # it will be released when session.commit is called.
+        session = self.SessionMaker()
+        session.connection(execution_options={"isolation_level": "SERIALIZABLE"})
```

Comment:
We don't need a serializable transaction for log_params and set_tags because we're only writing to a single table and we aren't performing any gap locking (e.g. `select for update`). Can we use `ManagedSessionMaker` instead?

Reply:
Done